src/patito/validators.py

```diff
@@ -256,6 +256,26 @@ def _find_errors(  # noqa: C901
                     )
                 )
 
+        # intercept struct columns, and get errors seperately
+        if schema.dtypes[column_name] == pl.Struct:
+            struct_errors = _find_errors(
+                dataframe=dataframe.select(column_name).unnest(column_name),
+                schema=schema.model_fields[column_name].annotation,
```

Comment:
Ok, so this part can be easily addressed using a try-except with `...annotation.__args__[0]` (or `...annotation.__args__[0].__args__[0]` if it is an optional list of structs). Now, I'm just getting some missing values issues, because I haven't passed back the nullable requirement.

Reply:
Perhaps, we could filter out rows in the submodel that have all-null columns before calling _find_errors again?

Reply:
Commit deb1493cdd24c11ba126209a95c729512583de51 filters out all-null rows via `dataframe.filter(pl.col(column_name).is_not_null())` before calling `_find_errors` again.